python/nav/alertengine/base.py

```diff
@@ -473,6 +390,94 @@ def process_single_queued_notification(queued_alert, now):
             return DISPATCH_FAILED
 
 
+def _verify_daily_dispatch(queued_alert, now, _logger=_logger):
```

Comment:
Gah, I want a refactoring checking tool (yes, this block was indeed moved from X to Y!)

Reply:
Again, I think `git diff` can be smarter about this, if you want to dive into things outside of the GitHub UI. I know I've at least used it to ignore simple whitespace changes, for a simpler use case.